community/contributors.md

```diff
@@ -82,4 +82,4 @@ us know in chat or team meeting.
   * Tihomir Surdilovic
 
 * **WSO2**
-  * Chathura Ekanayake
```

Comment:
This one is intended?

Reply:
Idk what happenned there, or what exactly has changed. 

I just reverted the change of removing @tsurdilo from contributors, as it is part of history and we should not rewrite it, as pointed out by @duglin.